sample-apps/django_sample/django_sample/settings.py

```diff
@@ -130,3 +130,33 @@
 # https://docs.djangoproject.com/en/4.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'formatters': {
+        'compact': {
+            'format': '{levelname} {message}',
+            'style': '{',
+        },
+    },
+    'filters': {
+        'require_debug_true': {
+            '()': 'django.utils.log.RequireDebugTrue',
+        },
+    },
+    'handlers': {
+        'console': {
+            'level': 'DEBUG',
+            'filters': ['require_debug_true'],
+            'class': 'logging.StreamHandler',
+            'formatter': 'compact'
+        },
+    },
+    'loggers': {
+        'django': {
+            'handlers': ['console'],
+            'propagate': True,
+        },
+    }
+}
```

Comment:
Does this just configure the "compact" logging? If so, I think it's probably unnecessary for this PR.

In the issue I'd asked for "Add logging to the sample app so we can reproduce the issue", and I should clarify that. What I mean is I'd like to see the sample app emit a custom log message (something like "hello world") so I can verify that there is no "[Judoscale]" prefix.

Right now the sample app just renders Django's default page, so I think it would be best to add a simple view for the root path (`/`) that just renders `<h1>Judoscale Sample App</h1>`. This view could contain the "hello world" log message mentioned above. Does this make sense?

Reply:
Yes I can do that.

Reply:
I'd like to add a test for this view too. Is that ok?